tools/liblz4/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2022 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lz4
+PKG_VERSION:=1.9.4
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/lz4/lz4/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=0b0e3aa07c8c063ddf40b082bdf7e37a1562bda40a0ff5272957f3e987e0e54b
+
+PKG_LICENSE:=BSD-2-Clause
+PKG_LICENSE_FILES:=LICENSE lib/LICENSE
+PKG_MAINTAINER:=
```

Comment:
Strictly speaking this isn't a package, and most builds in `tools/` lack `PKG_MAINTAINER`. I suggest to drop `PKG_MAINTAINER` field instead of setting it to an empty string.

Reply:
noted, thanks

I also tried to use `PKG_CONFIG_DEPENDS` which is apparently not wired for `tools`, figuring out a workaround for that.